config/Makefile.am

```diff
@@ -191,12 +191,17 @@ CONFIG_FILES = \
 	services/kpasswd.xml \
 	services/kprop.xml \
 	services/kshell.xml \
-	services/kube-api.xml \
```

Comment:
Dropping a service definition is a breaking change.

Consider making this an alias to `kubelet` via the `include` feature.

Reply:
Done